### PR TITLE
Off-ship Love

### PR DESCRIPTION
**Pull request by Hekzder**
Some love for off-ship roles like Colony and Survivor. Due to popular request and seeing first hand the immense pain that some skills can cause. 

[Current Skill Points for both Colony and Survivor are here.](https://gyazo.com/39700de04ee859b70ccbaab7778f9c2f)

[Proposed changes for Colony skill points are here.](https://gyazo.com/464043cb58ccbc29c701a10d66260ec7)

[Proposed changes for Survivor skill points are here.](https://gyazo.com/04599edafd6e82ee95b9e4be85b4e7a9)

Basically, TL;DR is: Colony loses 5 free skill points but gains 13 minimum skill points across various skills that make sense for a Colonist to have and make being a Colony doctor not immense pain. Survivor loses 3 free skill points but gains 9 minimum skill points, mainly just to make being a Survivor doctor not immense pain.

Other changes include some specialization based alt-titles for Colony to complement the fact that Colony doctors are now viable without being incredibly painful. An increase in one slot for both Colony and Ship Colony, since Colony has 5 beds and spawns but only 4 slots, and Ship Colony has 4 beds and spawns but only 3 slots. Also an equalization of the spawn weight of Colony and Ship Colony, both to 0.3, from 0.2 and 0.4 respectively. 

I'm willing to change pretty much anything in this PR based on feedback, more or less alt-titles, adjusted skills, etc. But I do think _something_ should be done about off-ship skills even if this is not **the exact** way to go.

### CODE REVIEW

**Comment by Hekzder** [comment]
I'll throw on a CL right before merge, in the event changes are suggested and such.

**Comment by Hekzder** [comment]
Okay [this](https://gyazo.com/567b676f0e0a67218ec1a0aaafbb325d) and [this](https://gyazo.com/0603d32e8dc912e51797c31c06d48a44) is kind of silly. With maintainer blessing I'd like to touch these skills and probably job slots as well, or I'll put it in a different PR if that isn't Kosher. Current max crew for Bearcat is 4 and that's kind of lame, Captain + 3 Crewman.

**Comment by BurpleBineapple** [comment]
Go ahead

**Comment by Hekzder** [comment]
> Go ahead

Neat. Will do so later tonight then.

**Comment by Hekzder** [comment]
Bearcat now gets some love as well.

[Captain SP adjustments](https://gyazo.com/e267a631eb26c198ab2c31e08e5c5dae) - -9 free skill points, +22 minimum skill points. (I don't count bureaucracy and finance, that was really just for fluff and because it makes sense)

[Crewman SP adjustments](https://gyazo.com/fb3412852767de0b4498efc5a136ab22) - -7 free skill points, +14 minimum skill points.

Bearcat has two more Crewman slots and alt-titles for specializations, bringing Bearcat's total members to 6, 5 crewmen and 1 Captain. Which lines up with the six beds in the crew cabin, since the actual original Captain of the Bearcat is dead and the acting Captain we see in game is the First Mate. TIL

Body scanners added to [Established Colony](https://gyazo.com/59836eac03d60b8bdfb1101343414c24) and [Bearcat](https://gyazo.com/d9d17d67502a3ffe68da89b983782816). [Two more voidsuits added to the Bearcat for the new crew members and one more jetpack.](https://gyazo.com/9408eb5490aa0f7792177ffa859e3ec7) 66% of the Bearcat crew now can have voidsuits, up from 50% and 50% can have jetpacks, the same as the previous 50%. 

This is now good for merge/review and has a CL.

**Review by Superhats — APPROVED** [review]
Good to go.

**Pull request merged**
